### PR TITLE
Switch to version Deluge 2.0.3

### PR DESCRIPTION
**Pull request by gepoul2003**
This repository has been quite helpful. I made some adjustments to build the 2.0.x release of Deluge.

### CODE REVIEW

File: deluge/Dockerfile [hunk 1]
```diff
@@ -2,31 +2,40 @@ FROM ubuntu:bionic
...
-    python-chardet \
-    geoip-database \ 
-    python-geoip \
-    python-libtorrent \
-    python-mako \
+    xz-utils \
+    build-essential \
+    libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev \
+    libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python3-tk \
+    libharfbuzz-dev libfribidi-dev libxcb1-dev \
```
Comment (Jalle19) [inline deluge/Dockerfile:29]: Are you sure all these GUI related dependencies are really required?

Comment (gepoul2003) [reply deluge/Dockerfile:29]: They are required to build [Pillow](https://pillow.readthedocs.io/en/latest/installation.html)

Comment (Jalle19) [reply deluge/Dockerfile:29]: Okay

Comment (Jalle19) [reply deluge/Dockerfile:29]: I tried building your branch and I got it to build without having to build Pillow. The new documentation on how to build Deluge 2.0 seems to be quite lacking.

Comment (gepoul2003) [reply deluge/Dockerfile:29]: I can see that you made the image quite smaller. I think we can close this PR.

**Pull request closed without merging**